packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -142,6 +149,7 @@ export const DataGrid: React.FC<GridProps> = (props) => {
         onRow={(record, recordIndex) => {
           return { onClick: () => onTapActionCallback(record, recordIndex) };
         }}
+        pagination={values?.hidePagination ? false : undefined}
```

Comment:
`values.hidePagination !== true`

Reply:
We can't update this condition that return the bool, as pagination only accept either false or undefined.